src/plugins/intel_cpu/src/cpu_memory.cpp

```diff
@@ -288,24 +288,28 @@ StaticMemory::StaticMemory(const dnnl::engine& eng, MemoryDescPtr desc, const vo
 
     m_size = m_pMemDesc->getCurrentMemSize();
 
-    auto dnnl_desc = MemoryDescUtils::convertToDnnlMemoryDesc(m_pMemDesc);
-
     if (data) {
         m_pMemMngr = std::make_shared<StaticMemoryMngr>(const_cast<void*>(data), m_size);
     } else {
         m_pMemMngr = std::make_shared<StaticMemoryMngr>(m_size);
     }
 
-    // ========================
-    // Equivalent of constructor memory(const primitive_desc &desc, void *hdl)
-    // but with ability to skip pads zeroing.
-    m_prim = memory(dnnl_desc->getDnnlDesc(), m_eng, DNNL_MEMORY_NONE);
-    //
-    // ========================
-    if (pads_zeroing)
-        m_prim.set_data_handle(m_pMemMngr->getRawPtr());
-    else
-        m_prim.set_data_handle_no_pads_proc(m_pMemMngr->getRawPtr());
+    try {
+        auto dnnl_desc = MemoryDescUtils::convertToDnnlMemoryDesc(m_pMemDesc);
+        // ========================
+        // Equivalent of constructor memory(const primitive_desc &desc, void *hdl)
+        // but with ability to skip pads zeroing.
+        m_prim = memory(dnnl_desc->getDnnlDesc(), m_eng, DNNL_MEMORY_NONE);
+        //
+        // ========================
+        if (pads_zeroing)
+            m_prim.set_data_handle(m_pMemMngr->getRawPtr());
+        else
+            m_prim.set_data_handle_no_pads_proc(m_pMemMngr->getRawPtr());
+    }
+    catch (const std::exception& exc) {
+        dnnlErrorCtx = exc.what();
+    }
```

Comment:
It does can handle "const->result" graph for unsupported precision, but it seems it cannot handle "const->convert->result" graph, such as:
`Constant(u16)->convert(u16->f32)->result(f32)`

Could you create such test cases?

Reply:
Aren't the enabled tests enough? Your proposed subgraph will likely be folded during the constant folding pass.